pkg/supportbundle/spec.go

```diff
@@ -727,6 +729,13 @@ func addDefaultDynamicTroubleshoot(supportBundle *troubleshootv1beta2.SupportBun
 func getDefaultDynamicCollectors(app *apptypes.App, imageName string, pullSecret *troubleshootv1beta2.ImagePullSecrets) []*troubleshootv1beta2.Collect {
 	collectors := make([]*troubleshootv1beta2.Collect, 0)
 
+	// Collect Cluster Resources for the namespace that the Admin Console is running in
```

Comment:
should this also include the app namespaces? i think we already have logic to determine what those namespaces are for minimal rbac that can be reused

Reply:
yeah, so some additional background is that we're making this change because a customer specifically wanted to exclude certain application namespaces from being collected and instead respect their instantiation of `clusterResources` which we weren't allowing before. 

But you do raise a good point. There might be some folks relying on the default behavior and not actually specifying all needed namespaces in their spec.